serialization/src/main/scala/sbt/serialization/Using.scala

```diff
@@ -0,0 +1,19 @@
+package sbt.serialization
+
+import java.io.{ File, FileOutputStream, BufferedWriter, OutputStreamWriter }
+import java.nio.charset.Charset
+
+object Using {
```

Comment:
forever not just for now - generic IO utilities don't belong in sbt serialization